editoast/src/models/scenario.rs

```diff
@@ -174,9 +187,15 @@ impl List<(i64, Ordering)> for ScenarioWithCountTrains {
     ) -> Result<PaginatedResponse<Self>> {
         let study_id = params.0;
         let ordering = params.1.to_sql();
-        sql_query(format!("SELECT t.*,COUNT(train_schedule.id) as trains_count FROM scenario as t
+        sql_query(format!("WITH scenarios_with_train_counts AS (
```

Comment:
I'm fine with the change but why do you want to add the infra name?

Reply:
It is required in the front. I’m not sure how it worked before. It is already returned in `ScenarioWithDetails`